components/renku_data_services/namespace/blueprints.py

```diff
@@ -141,3 +142,47 @@ async def _delete_member(_: Request, *, user: base_models.APIUser, slug: str, us
             return HTTPResponse(status=204)
 
         return "/groups/<slug>/members/<user_id>", ["DELETE"], _delete_member
+
+    def get_namespaces(self) -> BlueprintFactoryResponse:
```

Comment:
After changing the slug of a group, I am seeing it twice in the namespace listing (though it only appears once in the group listing).

Reply:
Re-tested. This seems to now cascade delete any projects in that group. Not 100% this is the behavior we want, but it is at least consistent; I will note in the dialog box that this action will delete all projects in that namespace.

Reply:
Re-tested. This works correctly now!